MovieApp/PresentationLayer/MovieDetails/CastMemberCell/CastMemberCell.swift

```diff
@@ -23,8 +23,11 @@ class CastMemberCell: UICollectionViewCell {
     }
 
     func setData(for castMember: CastMemberViewModel) {
-        let profileUrl = URL(string: castMember.profileSource ?? "")
-        profileImageView.kf.setImage(with: profileUrl)
+        if let profileUrl = URL(string: castMember.profileSource ?? "") {
+            profileImageView.kf.setImage(with: profileUrl)
```

Comment:
Kingfisher has a placeholder image option so you could use that instead of this if-else